pkg/resource/model_package_group/testdata/sdkapi/describe/deleting.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
[delete api](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.delete_model_package_group) doesnt return anything, you can just leave the output_fixture empty in test_suite and that should work

Reply:
this is a call to` - operation: DescribeModelPackageGroupWithContext` not to delete